index.html

```diff
@@ -964,6 +964,66 @@
   </div>
   <!-- ends Check interlacement   -->
 
+  <!-- QCTools Report -->
+  <span data-toggle="modal" data-target="#qctools"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="This will create q QCTools report for a video file with audio track">QCTools report (with audio)</button></span>
+  <div id="qctools" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Creates a QCTools report</h3>
+          <p><code>ffprobe -f lavfi -i "movie=<i>input_file</i>:s=v+a[in0][in1],[in0]signalstats=stat=tout+vrep+brng,cropdetect=reset=1:round=1,idet=half_life=1,split[a][b];[a]field=top[a1];[b]field=bottom,split[b1][b2];[a1][b1]psnr[c1];[c1][b2]ssim[out0];[in1]ebur128=metadata=1,astats=metadata=1:reset=1:length=0.4[out1]" -show_frames -show_versions -of xml=x=1:q=1 -noprivate | gzip > <i>input_file</i>.qctools.xml.gz</code></p>
+          <p>This will create an XML report for use in <a href="https://github.com/bavc/qctools">QCTools</a> for a video file with one video track and one audio track.</p>
+          <dl>
+            <dt>ffprobe</dt><dd>starts the command</dd>
+            <dt>-f lavfi</dt><dd>tells ffprobe to use the Libavfilter input virtual device <a href="http://ffmpeg.org/ffmpeg-devices.html#lavfi" target="_blank">[more]</a></dd>
+            <dt>-i</dt><dd>input file and parameters</dd>
+            <dt>"movie=<i>input_file</i>:s=v+a[in0][in1],[in0]signalstats=stat=tout+vrep+brng,cropdetect=reset=1:round=1,idet=half_life=1,split[a][b];[a]field=top[a1];[b]field=bottom,split[b1][b2];[a1][b1]psnr[c1];[c1][b2]ssim[out0];[in1]ebur128=metadata=1,astats=metadata=1:reset=1:length=0.4[out1]"</dt>
+            <dd>This very large lump of commands declares the input file and passes in a request for all potential data signal information for a file with one video and one audio track</dd>
+            <dt>-show_frames</dt><dd>asks for information about each frame and subtitle contained in the input multimedia stream</dd>
+            <dt>-show_versions</dt><dd>asks for information related to program and library versions</dd>
+            <dt>-of xml=x=1:q=1</dt><dd>sets the data export format to XML </dd>
+            <dt>-noprivate</dt><dd>hides any private data that might exist in the file</dd>
+            <dt>| gzip</dt><dd>The | is to "pipe" (or push) the data into a compressed file format</dd>
+            <dt><code>></code></dt><dd>redirects the standard output (the data made by ffprobe about the video) </dd>
+            <dt><i>input_file</i>.qctools.xml.gz</dt><dd>names the zipped data output file, which can be named anything but needs the extension qctools.xml.gz for compatibility issues</dd>
```

Comment:
No, I guess it’s meant to have the same name as the input file, but with `.qctools.xml.gz` appended which makes sense to me.